consensus/src/config.rs

```diff
@@ -78,27 +88,48 @@ pub fn exponential_slowdown(
 
 /// A default configuration of what the creators of this package see as optimal parameters.
 pub fn default_config(n_members: NodeCount, node_ix: NodeIndex, session_id: SessionId) -> Config {
-    let unit_creation_delay = Arc::new(|t| {
-        if t == 0 {
-            Duration::from_millis(5000)
-        } else {
-            exponential_slowdown(t, 500.0, 3000, 1.005)
-        }
-    });
-    let delay_config = DelayConfig {
-        tick_interval: Duration::from_millis(100),
-        requests_interval: Duration::from_millis(3000),
-        unit_rebroadcast_interval_min: Duration::from_millis(15000),
-        unit_rebroadcast_interval_max: Duration::from_millis(20000),
-        // 4000, 8000, 16000, 32000, ...
-        unit_creation_delay,
-        // 5000, 500, 500, 500, ... (till step 3000), 500, 500*1.005, 500*(1.005)^2, 500*(1.005)^3, ..., 10742207 (last step)
-    };
     Config {
         node_ix,
         session_id,
         n_members,
-        delay_config,
+        delay_config: DelayConfig {
+            tick_interval: Duration::from_millis(100),
+            requests_interval: Duration::from_millis(3000),
+            unit_rebroadcast_interval_min: Duration::from_millis(15000),
+            unit_rebroadcast_interval_max: Duration::from_millis(20000),
+            unit_creation_delay: default_unit_creation_delay(),
+            coord_request_delay: default_coord_request_delay(),
+            coord_request_recipients: default_coord_request_recipients(),
+        },
         max_round: 5000,
     }
 }
+
+/// 5000, 500, 500, 500, ... (till step 3000), 500, 500*1.005, 500*(1.005)^2, 500*(1.005)^3, ..., 10742207 (last step)
+fn default_unit_creation_delay() -> DelaySchedule {
+    Arc::new(|t| {
+        if t == 0 {
+            Duration::from_millis(5000)
+        } else {
+            exponential_slowdown(t, 500.0, 3000, 1.005)
+        }
+    })
+}
+
+/// 100, 1000, 3000, 6000, 9000, ...
+fn default_coord_request_delay() -> DelaySchedule {
+    Arc::new(|t| {
+        if t <= 1 {
```

Comment:
It seems to me that it is more like `100, 100, 1000, 3000, ...` -- because the first reschedule is with counter `0`?